inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/http/HttpBaseSource.java

```diff
@@ -48,64 +46,55 @@ public class HttpBaseSource extends AbstractSource implements EventDrivenSource,
     protected String attr;
     protected String messageHandlerName;
     protected boolean filterEmptyMsg;
-    protected CounterGroup counterGroup;
-    protected CounterGroupExt counterGroupExt;
     protected int maxConnections = Integer.MAX_VALUE;
     protected boolean customProcessor = false;
     protected Context context;
-    private int statIntervalSec;
-    private StatRunner statRunner;
-    private Thread statThread;
+    // statistic
+    protected MonitorIndex monitorIndex = null;
+    protected MonitorIndexExt monitorIndexExt = null;
+    private int statIntervalSec = 60;
+    private int maxMonitorCnt = 300000;
```

Comment:
the default value is set already so no need to declare here

Reply:
@EMsnap, this setting is necessary, regardless of whether the context() is called and whether the parameters are coded, these two parameters have data settings by default